Assets/XML/GlobalDefines.xml

```diff
@@ -187,7 +187,7 @@
 	</Define>
 	<Define>
 		<DefineName>MAX_YIELD_STACK</DefineName>
-		<iDefineIntVal>5</iDefineIntVal>
+		<iDefineIntVal>10</iDefineIntVal>

```

Comment:
What does this affect?

Reply:
It assigns 10 "slots" to each yield instead of the old 5. The new dds file consists of 32 slots of 64x64 images.